platform/linuxbsd/export/export.cpp

```diff
@@ -44,7 +44,7 @@ void register_linuxbsd_exporter() {
 	logo->create_from_image(img);
 	platform->set_logo(logo);
 	platform->set_name("Linux/X11");
-	platform->set_extension("x86");
+	platform->set_extension("x86_32");
```

Comment:
This is a significant change and it should not be sneaked into a refactoring PR.
It should have its own PR and discussion.

Reply:
This is purely a cosmetic change to the resulting executable, and it doesn't depend on or need changes in other parts of the code. I didn't see it as a significant change. But I removed it from this PR as you've asked.

Reply:
I think it's a good idea to change this for Godot 4.0; we just need to make sure to update the SteamDB heuristics.